### PR TITLE
Remove non fragmented data

### LINKED ISSUE #805: Remove non-fragmented data
**Is your feature request related to a problem? Please describe.**
Following the merge of https://github.com/BBC-News/simorgh/pull/783 we should remove the non-fragmented fixture data.

**Describe the solution you'd like**
Move onward journeys into `data/fragmented` then delete `data/test/` and `data/prod/` then rename `data/fragmented/` to `/data/`. There will be need to change the path the fixture data is loaded from https://github.com/BBC-News/simorgh/pull/783/files#diff-788dcef7a5e3a2740bf71979cca64a59R49. 

**Describe alternatives you've considered**
N/A

**Testing notes**
Will Cypress tests be required or are unit tests sufficient?

**Additional context**
N/A

- [x] Intially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #805 

_Replaces the current fixture data under `/data/test/` and `/data/prod/` with the data from `/data/fragmented/test/` and `/data/fragmented/prod/` respectively. Delete's the `fragmented/` directory. Removed `fragmented` from the `dataFolderToRender` path._

- ~[ ] Tests added for new features~
- [ ] Test engineer approval

NB: changes were made to `src/server/index.jsx` and `validateNode.test.jsx` however my have suppressed diff on GH due to the amount of files changes

### CODE REVIEW

**Comment by pjlee11** [comment]
blocked awaiting the merge of #783

**Comment by pjlee11** [comment]
unblocked

File: data/test/news/articles/c0000000011o.json [hunk 1]
```diff
@@ -128,26 +129,31 @@
...
-                "blockId": "a39cecf2-153a-eaa1-2ff4-358d3ac35912",
                 "type": "paragraph",
                 "model": {
-                  "text": "Marks and Spencer plans to close 100 stores by 2022, accelerating a reorganisation that it says is \"vital\" for the retailer's future."
+                  "text": "Marks and Spencer plans to close 100 stores by 2022, accelerating a reorganisation that it says is \"vital\" for the retailer's future.",
+                  "blocks": [
+                    {
+                      "type": "fragment",
+                      "model": {
+                        "text": "Marks and Spencer plans to close 100 stores by 2022, accelerating a reorganisation that it says is \"vital\" for the retailer's future.",
```
Comment (bcmn) [inline data/test/news/articles/c0000000011o.json:142]: Is the `\"vital\"` here supposed to be an attribute instead?

File: data/test/news/articles/c0000000012o.json [hunk 2]
```diff
@@ -199,27 +199,32 @@
...
-                "blockId": "89cee09e-cb72-708a-074e-323d3469dab1",
                 "type": "paragraph",
                 "model": {
-                  "text": "Marks and Spencer plans to close 100 stores by 2022, accelerating a reorganisation that it says is \"vital\" for the retailer's future."
+                  "text": "Marks and Spencer plans to close 100 stores by 2022, accelerating a reorganisation that it says is \"vital\" for the retailer's future.",
+                  "blocks": [
+                    {
+                      "type": "fragment",
+                      "model": {
+                        "text": "Marks and Spencer plans to close 100 stores by 2022, accelerating a reorganisation that it says is \"vital\" for the retailer's future.",
```
Comment (bcmn) [inline data/test/news/articles/c0000000012o.json:212]: See above comment.

File: data/test/news/articles/c0000000016o.json [hunk 3]
```diff
@@ -125,41 +133,62 @@
...
-                "blockId": "6210d343-1825-4ac6-7f58-94a9b2583921",
                 "type": "paragraph",
                 "model": {
-                  "text": "Nam luctus libero quis leo sodales, quis blandit eros commodo. Mauris quam tortor, \"porttitor vitae nunc sit amet\", tincidunt molestie metus."
+                  "text": "Nam luctus libero quis leo sodales, quis blandit eros commodo. Mauris quam tortor, \"porttitor vitae nunc sit amet\", tincidunt molestie metus.",
+                  "blocks": [
+                    {
+                      "type": "fragment",
+                      "model": {
+                        "text": "Nam luctus libero quis leo sodales, quis blandit eros commodo. Mauris quam tortor, \"porttitor vitae nunc sit amet\", tincidunt molestie metus.",
```
Comment (bcmn) [inline data/test/news/articles/c0000000016o.json:146]: See above.

File: data/test/news/articles/c0000000011o.json (hunk 1 above)
Comment (bcmn) [reply data/test/news/articles/c0000000011o.json:142]: Never mind, it's escape characters. 😅

File: data/test/news/articles/c0000000005o.json [hunk 4]
```diff
@@ -134,77 +165,33 @@
...
-          "type": "image",
-          "model": {
-            "blocks": [
-              {
-                "blockId": "13097b46-67c7-cbe6-5777-63088d479fea",
-                "type": "rawImage",
-                "model": {
-                  "width": 1024,
-                  "height": 576,
-                  "locator": "/cpsprodpb/D6C0/production/_101767945_gettyimages-519045912.jpg",
```
Comment (bcmn) [inline data/test/news/articles/c0000000005o.json:188]: Is there any reason this specific image has been removed?

Comment (pjlee11) [reply data/test/news/articles/c0000000005o.json:188]: The diff is misleading, I haven't edited these files but replaced them with the fragmented payloads. I can check with @jimjohnsonrollings if he meant to remove this particular image

Comment (jimjohnsonrollings) [reply data/test/news/articles/c0000000005o.json:188]: I think that's just a mistake on my part - I've accidently removed an image block and a text block.

Comment (pjlee11) [reply data/test/news/articles/c0000000005o.json:188]: Restored the text, image and text blocks

Comment (pjlee11) [reply data/test/news/articles/c0000000005o.json:188]: FYI the locator was incorrectly prefixed with `/cpsprodpb/` so also removed that

**Review by bcmn — APPROVED** [review]

**Review by jtart — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**